### DISCUSSION TITLE
Questions about Farmer's Delight + Botany Bots Integration

### DISCUSSION THREAD

**Discussion by Kaligraphy22** [vectorwing/FarmersDelight#607] [Q&A]
I remember Botany Pots' author saying that the issue with rich soil not working properly in the pots was on Farmer's Delight. Did that ever get resolved? Also, onions and rice don't work in the pots while tomatoes and cabbage do. Was that resolved? I checked the issues:open and issues:closed but didn't see anything.  I don't know if this issue was lumped into another issue and I want to avoid posting duplicate issues.

**Comment by vectorwing** [comment]
@Kaligraphy22, sorry for the delay! This got resolved in FD 1.2.1, I believe.

Darkhax moved the Botany Pots integration within the mod itself, so now it inherently supports FD. From my end, version 1.2.1 removed the integration so it doesn't cause bugs.

Please update both mods, the compat should be fixed by then. 👍
